desc/equilibrium/equilibrium.py

```diff
@@ -1221,6 +1221,43 @@ def map_coordinates(
             **kwargs,
         )
 
+    def get_rtz_grid(
```

Comment:
initially we decided this wouldn't be a method of `Equilibrium`, just a standalone function. I see it's both here and in some of the PRs by @unalmis. I would slightly prefer it not be part of the class since for most purposes the user just needs to create a normal grid with `coordinates="raz"` and the transformation happens automatically, but I'm open to keeping it here

Reply:
Let's change it elsewhere.

Reply:
The reason this was made an object method is to avoid the circular import errors that occur if you import the function directly from `coords`